Translators/WZDx/geospatial_service.py

```diff
@@ -0,0 +1,177 @@
+import os
+import requests
+import urllib3
+urllib3.disable_warnings()
+
+
+def get_geospatial_endpoint():
+    return os.environ['dual_carriageway_endpoint']
+
+
+def get_map_server_endpoint():
+    return os.environ['dual_carriageway_endpoint'].split('/exts', 1)[0]
+
+
+def measure_at_point(lat, lon, routeId) -> int:
+    '''Finds the measure at a given point for a given route
+
+    Parameters:
+        lat (float): Latitude of point
+        lon (float): Longitude of point
+        route (str): Name of route
+
+    Returns:
+        measure (int): Measure of point on route'''
+    r = requests.get(
+        f'{get_geospatial_endpoint()}/MeasureAtPoint?x={lon}&y={lat}&inSR=4326&routeId={routeId}&tolerance=&outSR=4326&f=pjson', verify=False)
+    data = r.json()
+    return int(data["features"][0]["attributes"]["Measure"])

```

Comment:
Could be useful to add a check here to make sure the API returned a valid message, since it is failing so much

Reply:
I've added a check for 200 status code as well as a schema validation in there to verify before parsing. Let me know what you think

Reply:
Looks good!